### DISCUSSION TITLE
How can I get my app's icon to show up when sixel support is enabled

### DISCUSSION THREAD

**Discussion by rkeithhill-keysight** [microsoft/winget-cli#4986] [Q&A] [answered]
I'm looking at manifests for apps that show a sixel image and I don't see where winget is pulling that image.  There's no manifest entry for a URL to an icon.  Is it pulling the image from the installer exe?  If so, our installer is based on Installer Builder and winget doesn't show a sixel image for it - even though I can see our app icon on the exe in File Explorer.  Does the app icon have to provide a certain size icon or what?

**Comment by SpecterShell** [comment, marked as answer]
The icons were automatically extracted from the executables installed by the installer during automatic validation and then compiled with the (compressed) manifests stored on the WinGet index server.

The automatic icon extraction was enabled recently (so manifests prior to this date don't have icons) and does not work on every installer though.

**Reply by SpecterShell** [reply]
Take `PawelSalawa.SQLiteStudio` as an example, which is InstallBuilder-based and the latest version was added some days ago.
![图片](https://github.com/user-attachments/assets/30c62327-e84a-416b-b498-859df74a4234)

**Reply by rkeithhill-keysight** [reply]
Thanks!  Our icon is now showing up. Woohoo! 
![image](https://github.com/user-attachments/assets/d797f3fb-b4f8-4273-b162-db7e1796962c)
